bro_connector/frd/models.py

```diff
@@ -290,4 +290,31 @@ class FormationresistanceRecord(FRDRecord):
     class Meta:
         managed = True
         db_table = 'frd"."formationresistance_record'
-        verbose_name_plural = "Formationresistance Records"
\ No newline at end of file
+        verbose_name_plural = "Formationresistance Records"
+
+class FrdSyncLog(models.Model):
+    synced = models.BooleanField(default=False)
+    date_modified = models.CharField(max_length=254, null=True, blank=True)
+    event_type = models.CharField(
+        choices=EVENT_TYPE_CHOICES,
+        blank=False,
+        null=True,
+        max_length = 25,
+    )
+    frd = models.ForeignKey(FormationResistanceDossier, on_delete = models.CASCADE, null = True, blank = True)
+    frd_bro_id = models.CharField(max_length=254, null=True, blank=True)
+    process_status = models.CharField(max_length=254, null=True, blank=True)
+    comment = models.CharField(max_length=10000, null=True, blank=True)
+    xml_filepath = models.CharField(max_length=254, null=True, blank=True)
+    delivery_status = models.IntegerField(choices=LEVERINGSTATUS_CHOICES, null=True, blank=True, default = 0)
+    delivery_status_info = models.CharField(max_length=254, null=True, blank=True)
+    delivery_id = models.CharField(max_length=254, null=True, blank=True)
+    
+        
+    def __str__(self):
+        return f"{self.event_type}_{self.frd.object_id_accountable_party}_log"
+
+    class Meta:
+        db_table = 'frd"."frd_sync_log'
+        verbose_name = "FRD Synchronisatie Log"
+        verbose_name_plural = "FRD Synchronisatie Logs"
```

Comment:
Ik zou in het Nederlands gewoon de reguliere spelling aanhouden, dus Zonder Hoofdletters Bij Elk Woord Want Wij Zijn Geen Amerikanen.
En ik vind dat "synchronisatielog" taalkundig aan elkaar hoort :-)